gradle.properties

```diff
@@ -0,0 +1,5 @@
+org.gradle.daemon=true
```

Comment:
I thought daemon was already the default. It seems to be the default on my machine.